source/main/utils/HandleDiskSpace.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+import { BrowserWindow, ipcMain } from 'electron';
+import checkDiskSpace from 'check-disk-space';
+import prettysize from 'prettysize';
+import { GET_DISK_SPACE_STATUS } from '../../common/ipc-api';
+import { environment } from '../environment';
+import { Logger } from './logging';
+import {
+  DISK_SPACE_REQUIRED,
+  DISK_SPACE_REQUIRED_MARGIN_PERCENTAGE,
+  DISK_SPACE_CHECK_LONG_INTERVAL,
+  DISK_SPACE_CHECK_SHORT_INTERVAL,
+  DISK_SPACE_RECOMMENDED_PERCENTAGE
+} from '../config';
+
+export default (
```

Comment:
I think we agreed on always using explicit exports like `export const handleDiskSpace = …` (because this makes refactoring tasks easier, since the default export can be named arbitrarily in the import location and thus refactoring tools do not pick that up correctly)

Reply:
✅